<launch>
  <!-- Arguments -->
  <arg name="cmd_vel_topic" default="robot3/cmd_vel" />
  <arg name="odom_topic" default="robot3/odom" />
  <arg name="scan_topic" default="robot3/scan" />

  <!--node pkg="arobota2" type="omni_cmdveltf.py" name="Vel_tf" output="screen"/!-->
  <!-- move_base -->
  <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
    <param name="base_local_planner" value="dwa_local_planner/DWAPlannerROS"/>
    <rosparam file="$(find arobota2)/param/common_costmap.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find arobota2)/param/common_costmap.yaml" command="load" ns="local_costmap" />
    <rosparam file="$(find arobota2)/param/local_costmap_params.yaml" command="load" />
    <rosparam file="$(find arobota2)/param/global_costmap_params.yaml" command="load" />
    <rosparam file="$(find arobota2)/param/movebase_params.yaml" command="load" />
    <rosparam file="$(find arobota2)/param/dwa_local_planner_params.yaml" command="load" />
    <rosparam file="$(find arobota2)/param/global_costmap_plugin_config.yaml" command="load" /> 
    <rosparam file="$(find arobota2)/param/local_costmap_plugin_config.yaml" command="load" />
    <param name="global_costmap/robot_base_frame" value="robot3/base_footprint"/>
    <param name="local_costmap/robot_base_frame" value="robot3/base_footprint"/>
    <param name="local_costmap/global_frame" value="robot3/odom"/>


    <!--remap from="cmd_vel" to="$(arg cmd_vel_topic)"/>
    <remap from="odom" to="$(arg odom_topic)"/-->
    <remap from="map" to="/map"/>
  </node>
</launch>
